extensions/wikia/AdEngine/js/spec/lookup/adapter/appnexusPlacements.spec.js

```diff
@@ -0,0 +1,69 @@
+describe('ext.wikia.adEngine.lookup.adapter.appnexusPlacements', function () {
+	'use strict';
+
+	var mocks = {
+		adLogicZoneParams: {
+			getVertical: function() {}
+		},
+		window: {
+			location: {
+				host: ''
+			}
+		}
+	}, testCases = [
+		{
+			vertical: 'lifestyle',
+			skin: 'mercury',
+			host: 'http://muppet.wikia.com',
+			expected: '9412994'
```

Comment:
same as https://github.com/Wikia/app/pull/11269/files#r77587392. I don't have a strong opinion however I don't like to add variables where it's not necessary (no chance for re-usage)